### PR TITLE
Init landing page

### PR DESCRIPTION
**Pull request by GraceMattern**
Changes to file structure/use
- did not use pages directory but instead components to load elements of landing page
- did not use App directory
- created app.tsx in src as main launcher for landing

Added 
- project name 
- blurb
- companies you can track svg
- login/signup button (no redirect... yet!)
- logo svg
- global and styleguide CSS

### CODE REVIEW

**Comment by roccopolimen** [comment]
* Why did you not use the landing page component or the app page component? The way I set up the code was so that the only non-directory file in the src/ folder was index.tsx since that is the entry point.
* Take advantage of Material UI instead of building your pieces from scratch.
* styles/ is the place where global styles should be going if they are needed beyond what material ui components provide.
* Each component/page should be a folder with the component name that contains an index.tsx and a styles.css(optional)
i.e.:
Landing/
 -> index.tsx
 -> [style.css]

File: client/src/app.tsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import React from "react";
```
Comment (roccopolimen) [inline client/src/app.tsx]: don't import React where we don't use it

File: client/src/app.tsx [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+import { BrowserRouter as Router, Route } from "react-router-dom";
+import "./app.css";
+import "./styleguide.css"
+import { Landing } from "./components/landing/Landing"
```
Comment (roccopolimen) [inline client/src/app.tsx]: use absolute pathing, that is the point of the tsconfig.json file I created

File: client/src/components/landing/Landing.tsx [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+                <div className="landing-page screen">
+                    <div className="landing-design-v1">
+                    <div className="nav-bar">
+                        {/* TODO put logo here */}
+                        {/* TODO maybe put a login hyperlink */}
+                    </div>
+                    <div className="content">
+                        <div className="heading-logo">
+                        <div className="heading-cta">
+                            <div className="heading-text">
```
Comment (roccopolimen) [inline client/src/components/landing/Landing.tsx]: Use MUI for this so we have a consistent style https://mui.com/components/

**Review by roccopolimen — COMMENTED** [review]
Small changes on top of the changes noted in the conversation section of this pull request

**Comment by GraceMattern** [comment]
- I didn't use the original file structure at first because the landing page wouldn't load. I found out it was the App/index.tsx Themeprovider block. I'm not sure if that is important for the landing page but commenting it out allows the landing page to load now
- I tried to do the absolute paths in tsconfig but it wouldn't work. I'm not sure if I did it correctly but I've commented it out and you can see
- I used material UI for the button but I'm not sure if you wanted it elsewhere. It is good for consistency but not custom control

**Comment by roccopolimen** [comment]
@GraceMattern I pushed a small commit to show you how it will work with the current structure of the code. Let me know if you have any questions about the structure, but I'll probably go over it with everyone on Wednesday so we are all on the same page.

Other than that things look pretty good with this 👍

**Comment by roccopolimen** [comment]
> * I used material UI for the button but I'm not sure if you wanted it elsewhere. It is good for consistency but not custom control

I personally rely on it a lot cuz I hate CSS, but I would say in terms of controlling the screen layout, it is not absolutely necessary if you prefer to do CSS + div, but in terms of styling components I think it is necessary

**Comment by GraceMattern** [comment]
I looked over the changes you made
- Why did you remove the theme in Landing/index.tsx? The button is now the standard material UI blue instead of the purple on the figma. Did you want to have the theme defined in /constants
- The changes also seemed to make the background black instead of the purple-white color (easy fix though)
- Why did you remove the ../ in import styles/styleguide.css in App/index.tsx

**Comment by roccopolimen** [comment]
@GraceMattern thats my bad! I was going to move the theme style over to the constants/ folder, there is currently a basic dark theme style in there. You can just overwrite that style with the palette you were using. Basically I just wanted to make the style global to the whole site, and that theme is used inside of the App component so it gets inherited to its children

**Comment by roccopolimen** [comment]
Okay I applied the theme globally to the website through the constants

Also, I noticed the Landing page has too much padding I believe because it does not fit centered into the window and I have to scroll to see it. You should look into using the MUI Box component for centering things on the page, and also the Typography component for putting text into it instead of html elements purely.

**Comment by GraceMattern** [comment]
Changes 456557d and b61b41f are more like one commit but I didn't know you were working on the code at the same time as me so they were committed separately

**Comment by GraceMattern** [comment]
> Also, I noticed the Landing page has too much padding I believe because it does not fit centered into the window and I have to scroll to see it. You should look into using the MUI Box component for centering things on the page, and also the Typography component for putting text into it instead of html elements purely.

I'm not sure what you mean since it looks ok on my end. We can discuss it on Wednesday

**Comment by GraceMattern** [comment]
Known issues:
- amazon logo gets stretched at a certain window size 
- buttons are not purple (constant/theme)

File: client/src/pages/Landing/index.tsx [hunk 4]
```diff
@@ -1,10 +1,92 @@
 import './style.css';
+import {Grid, Slide, Typography, useMediaQuery, Button } from '@material-ui/core';
```
Comment (mpolimen) [inline client/src/pages/Landing/index.tsx]: Don't think useMediaQuery is ever used

File: client/src/pages/Landing/index.tsx [hunk 5]
```diff
@@ -1,10 +1,92 @@
...
-        <div>Hello World</div>
-    );
+        {/* bottom grid */}
+        <Grid container item sm={12} lg={9} justify='center' alignItems='center' spacing={3}>
+          {/* body 2 left */}
+          <Grid item sm={12} lg={6}>
+            <Slide in={true} direction='up' timeout={2000}>
+              <div className='HomePageContainer'>
+                <Typography variant='h5'>Track Applications for Top Companies</Typography>
+                <div className="logos">
```
Comment (mpolimen) [inline client/src/pages/Landing/index.tsx:76]: The logo spacing and all doesn't look great, especially when resizing (as you noted), but I don't see a dire need for adjustment at this stage; we'll just keep it in mind.

File: client/src/pages/Landing/style.css [hunk 6]
```diff
@@ -1,2 +1,73 @@
 /* TODO: this entire file is optional, just showing you it can be here */
-/* Use this to add css styles to the Landing React Element */
\ No newline at end of file
+/* Use this to add css styles to the Landing React Element */
+
```
Comment (mpolimen) [inline client/src/pages/Landing/style.css:3]: On the figma, I used **Poppins** as my font; import it from google fonts or maybe grab something different. I don't love the current font. Maybe add it "globally" in src/App/style.css as the default font to keep the whole site consistent, when possible

Comment (GraceMattern) [reply client/src/pages/Landing/style.css:3]: You don't like helvetica? it's a classic

**Comment by GraceMattern** [comment]
Changes made:
- change logo svg to icons because it was sooo annoying to look kinda ok :/
- import Poppins font
- apply Poppins globally (app/style, constant theme)
- remove styleguide.css
- landing, signin, signup fonts

**Comment by mpolimen** [comment]
LGTM 👍

**Comment by roccopolimen** [comment]
Great Stuff! Merging...

**Pull request merged**